crates/topos-tce-api/src/stream/mod.rs

```diff
@@ -150,14 +150,25 @@ impl Stream {
                     }
                 }
 
-                Some(_stream_packet) = self.inbound_stream.next() => {
-
+                // We currently open the stream, but no other message from the client is getting processed.
+                // We are using this open connection to communicate `delivered_certificates` to the client.
+                Some(stream_packet) = self.inbound_stream.next() => {
+                    match stream_packet {
+                        Ok((_request_id, _message)) => {
+                            debug!("Received message from stream_id: {:?}", self.stream_id);
```

Comment:
Should probably be `trace`?

Reply:
Do we have any good way of correlating a stream_id with the actual nodes involved? If not, is it still useful?

Reply:
I used it to test locally and thought I leave it in here to explain what happens, logically. We don't send any message to the stream from the client and if so, do nothing with it. Maybe removing the `log` all together and leaving a comment is a better choice.

Reply:
A `trace` is also fin for me. Your call!